ui/app/src/App.tsx

```diff
@@ -3,17 +3,16 @@ import { useState } from "react";
 import { Stream } from "@apeworx/apepay";
 import { TokenInfo } from "@uniswap/token-lists";
 import { ConnectButton } from "@rainbow-me/rainbowkit";
-
-// import { CreateStream, StreamStatus } from "@apeworx/apepay-react";
-import { StreamStatus } from "@apeworx/apepay-react";
-import CreateStream from "../../../ui/lib/CreateStream";
 import config from "./config";
 // NOTE: Do this or else it won't render (or create your own CSS)
 import "rc-slider/assets/index.css";
 import "./styles.css";
+import CreateStream from "lib/CreateStream";
+import StreamStatus from "lib/StreamStatus";
```

Comment:
Would prefer a solution here that mirrors line 3 above and uses the package name  (but allows dynamically updating those packages as long as they are installed in a specific way for development)